cpp/src/io/orc/reader_impl.cu

```diff
@@ -379,7 +379,7 @@ rmm::device_buffer reader::impl::decompress_stripe_data(
     device_span<device_span<uint8_t>> inflate_out_view{inflate_out.data(), num_compressed_blocks};
     switch (decompressor.compression()) {
       case compression_type::ZLIB:
-        if (nvcomp_integration::is_all_enabled()) {
+        if (false) {
```

Comment:
Should this (and change above) be documented/commented with a tracking issue so we will change back later?

Reply:
Added comments with the issue link, thanks for the suggestion! :+1: